simplified-app-ekirjasto/src/main/java/fi/kansalliskirjasto/ekirjasto/EkirjastoDocumentStoreConfiguration.kt

```diff
@@ -16,7 +18,7 @@ class EkirjastoDocumentStoreConfiguration : DocumentConfigurationServiceType {
   override val feedback: DocumentConfiguration? =
     DocumentConfiguration(
       name = null,
-      remoteURI = URI.create("https://lib.e-kirjasto.fi/palaute/?lang=__LANGUAGE__")
+      remoteURI = URI.create("http://10.0.2.2:5000/palaute/?lang=__LANGUAGE__&device_model=${Build.MANUFACTURER}%20${Build.MODEL}&software_version=${BuildConfig.VERSION_NAME}%20(${BuildConfig.VERSION_CODE})")
```

Comment:
Is 10.0.2.2 intentionally left here? On Android, this is the "host loopback interface", i.e. your development machine's localhost (127.0.0.1 on your development machine). I don't think this would work in production (or on other developers machines, unless they have this service running)?

It would be nice if this was configurable (possible to override) through local.properties, so passed through BuildConfig like `VERSION_NAME`, etc. That way, it would be easy to override for you (and we could even have a separate feedback service in lib-beta, etc., and the address could be set for each build flavor).

Reply:
No, pure mistake, good thing you caught it. I use it to test the locally run server. Send me the local.properties settings and I'll see if that works.

Reply:
I now made the change

Reply:
It's now in the backlog as EKIR-207.